src/main/java/de/tum/in/www1/artemis/web/rest/repository/TestRepositoryResource.java

```diff
@@ -104,7 +104,7 @@ boolean canAccessRepository(Long exerciseId) {
      * @throws IOException
```

Comment:
can the `@throws IOException` Comment be removed?

Reply:
I want to do the doc updates for this file in another PR.